library/ssl_tls12_server.c

```diff
@@ -4094,7 +4097,7 @@ static int ssl_parse_certificate_verify( mbedtls_ssl_context *ssl )
 {
     int ret = MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE;
     size_t i, sig_len;
-    unsigned char hash[48];
+    unsigned char hash[HASH_MAX_SIZE];
```

Comment:
I can't see why this is necessary either: this buffer is only written in the calc_verify() operation which only can be SHA384 or SHA256.

Reply:
https://github.com/Mbed-TLS/mbedtls/pull/6153#discussion_r934794060